dotnet-desktop-guide/framework/winforms/controls/numericupdown-control-windows-forms.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "NumericUpDown Control"
+description: Learn about the NumericUpDown control in Windows Forms, which like a combination of a text box and a pair of arrows that the user can click to adjust a value.
```

Comment:
Missed a word--I believe you meant "which looks like"

Reply:
Addressed.